lib/convert-to-amd-filter.js

```diff
@@ -127,11 +128,6 @@ module.exports = class ConvertToAMD extends Filter {
 
     // Replace the original ember scripts by the amd ones
     scriptElements.remove();
-    const amdScripts = [
```

Comment:
In the broccoli-filter pass of files, we now just catalog the scripts and remove them.  We will inject the necessary new scripts later in `build`